### ISSUE TITLE
images do not match error in the pad function?

### ISSUE THREAD

**Issue by eduOS** [lukas-blecher/LaTeX-OCR#76]
It seems that the sizes of the images don't match. Anyone else encountering this error?

```
Traceback (most recent call last):
  File "pix2tex.py", line 198, in <module>
    pred = call_model(args, *objs, img=img)
  File "pix2tex.py", line 84, in call_model
    input_image = pad(img).convert('RGB').copy()
  File "/mnt/lerner/ocr_latex/LaTeX-OCR/utils/utils.py", line 104, in pad
    padded.paste(im, im.getbbox())
  File "/opt/conda/lib/python3.8/site-packages/PIL/Image.py", line 1527, in paste
    self.im.paste(im, box)
ValueError: images do not match
```

**Comment by lukas-blecher** [comment]
I did come across this error before but I'm not sure what the reason was.
Which image causes this error?

**Comment by eduOS** [comment]
It only happens for an image with the following outputs: 

```
    print(im.size, padded.size, im.getbbox())
    # (672, 78) (672, 96) (0, 0, 661, 77)
```

**Comment by yht4work** [comment]
encounter same issue, here is the trigger.

<img width="818" alt="Snipaste_2022-04-01_14-57-13" src="https://user-images.githubusercontent.com/92348729/161235857-feeb7151-f737-43cd-a7f2-64bb78882be4.png">

**Comment by lukas-blecher** [comment]
Possibly resolved in #112 
Since I wasn't able to replicate the error I'm closing the issue now. 
Feel free to reopen if the issue persists
